server/index.ts

```
@@ -561,6 +754,93 @@ const extractDocumentContent = (ydoc: Y.Doc): string => {
+    let underlineCount = 0;
+    let codeCount = 0;
+    
+    const analyzeElement = (element: Y.XmlElement | Y.XmlFragment): void => {
+      if (element instanceof Y.XmlElement) {
+        switch (element.nodeName) {
+          case 'textStyle':
+            textStyleCount++;
+            const attrs = element.getAttributes();
+            console.log(`🎨 TextStyle attributes:`, attrs);
+            break;
+          case 'highlight':
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)